### PR TITLE
Always show tutanota logo in about dialog

### LINKED ISSUE #3666: About dialog shows whitelabel logo instead of Tutanota
At the top of the about dialog you see "Powered by" and then the Tutanota logo. If a whitelabel theme is configured with a custom logo, then that logo will be shown rather than the Tutanota one. 

This should always show the Tutanota logo, regardless of whitelabel configuration.

### PR DESCRIPTION
**Pull request by johnbotris**
fix #3666

### CODE REVIEW

File: src/gui/theme.js [hunk 1]
```diff
@@ -103,3 +105,7 @@ export function getNavigationMenuBg(): string {
 export function getNavigationMenuIcon(): string {
 	return theme.navigation_menu_icon || theme.navigation_button_icon
 }
+
+export function getColouredTutanotaLogo(): string {
+	return getLogoSvg(theme.content_accent, isColorLight(theme.content_bg) ? '#4a4a4a' : '#c5c7c7')
```
Comment (charlag) [inline src/gui/theme.js:110]: Don't hardcode colors, you can get the theme you need and get the color from there

**Review by charlag — CHANGES_REQUESTED** [review]

File: src/gui/theme.js (hunk 1 above)
Comment (johnbotris) [reply src/gui/theme.js:110]: these values don't exist on the theme but we can just grab the constants

Comment (charlag) [reply src/gui/theme.js:110]: yeah that would be okay

File: src/settings/AboutDialog.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ export class AboutDialog implements MComponent<void> {
 	view(vnode: Vnode<void>): Children {
 		return m(".flex.col", [
 			m(".center.mt", "Powered by"),
-			m(".center.mt", m.trust(theme.logo)),
+			m(".center.mt", m.trust(getColouredTutanotaLogo())),
```
Comment (charlag) [inline src/settings/AboutDialog.js:19]: This might do a lot of string interpolation but should be okay for about dialog

**Review by charlag — APPROVED** [review]

**Pull request merged**